examples/react/components/Content.tsx

```diff
@@ -181,13 +181,21 @@ const Content: React.FC = () => {
     }
   };
 
+  interface InputElements extends EventTarget {
+    elements: {
+      fieldset: HTMLFieldSetElement;
+      message: HTMLInputElement;
+      donation: HTMLInputElement;
+      multiple: HTMLInputElement;
+      button: HTMLButtonElement;
+    };
+  }
+  
```

Comment:
While this fixes the issue for the current elements of the form, if in the future we add a new element to the form then we would need to update this interface too.

I would suggest that we use a more "generic" type here, declare it at the top of the `Context.tsx` file below other imports:

```ts
export type Submitted = SubmitEvent & {
  target: { elements: { [key: string]: HTMLInputElement } };
};
```

Then we will need to make these changes (line 195):

```ts
async (e: Submitted ) => {}
```

And then align it in the `onSubmit` prop of the form

```jsx
<Form
  account={account}
  onSubmit={(e) => handleSubmit(e as unknown as Submitted)}
/>
 ```

Reply:
I don't know if the above generic solution will work for this case as fieldset is set as an "HTMLFieldSetElement".

Is there a way to add this into the elements? Would this also assume that all future elements will strictly be HTMLInputElements?

Reply:
Well, the reason `HTMLInputElement ` would work fine, here is that in the code we access only these properties: `disabled`, `value`,  `checked`, `focus()` looking at the docs here these properties will be present: https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement so typescript would not complain.

Since the goal of this PR is to fix the types for the current form elements and I like the idea of setting the correct types I am going to approve it.